relentless/simulate/hoomd.py

```diff
@@ -61,26 +71,32 @@ def extract_box_params(self, sim):
         Raises
         ------
         ValueError
-            If the volume is not set.
+            If the extent is not set.
         TypeError
-            If the volume does not derive from :class:`~relentless.volume.TriclinicBox`.
+            If the extent does not derive from :class:`~relentless.extent.TriclinicBox` or :class:`~relentless.extent.ObliqueArea`.
 
         """
         # cast simulation box in HOOMD parameters
         V = sim.ensemble.V
         if V is None:
-            raise ValueError('Box volume must be set.')
-        elif not isinstance(V, TriclinicBox):
-            raise TypeError('HOOMD boxes must be derived from TriclinicBox')
-
-        Lx = V.a[0]
-        Ly = V.b[1]
-        Lz = V.c[2]
-        xy = V.b[0]/Ly
-        xz = V.c[0]/Lz
-        yz = V.c[1]/Lz
-
-        return numpy.array([Lx,Ly,Lz,xy,xz,yz])
+            raise ValueError('Box extent must be set.')
+        elif not isinstance(V, (TriclinicBox, ObliqueArea)):
+            raise TypeError('HOOMD boxes must be derived from TriclinicBox or ObliqueArea')
+        if isinstance(V, TriclinicBox):
```

Comment:
Add a little whitespace so ifs don't look like they go together (might be my fault)
```suggestion
        
        if isinstance(V, TriclinicBox):
```